### PR TITLE
Fix check style errors

### PR DESCRIPTION
**Pull request by MichielvdBerg**

### CODE REVIEW

**Comment by MichielvdBerg** [comment]
This branch has merge conflicts due to an earlier merge, i will first resolve this

File: src/test/java/lasersharks/controllers/AudioControllerTest.java [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
-
+import javafx.embed.swing.JFXPanel;
+import lasersharks.Options;
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
 
-import javafx.embed.swing.JFXPanel;
-import lasersharks.Options;
+import static org.junit.Assert.*;
```
Comment (Exclaminator) [inline src/test/java/lasersharks/controllers/AudioControllerTest.java]: Please specify imports.

File: src/test/java/lasersharks/controllers/DirectionInputControllerTest.java [hunk 2]
```diff
@@ -16,15 +13,14 @@
...
-import javafx.scene.Scene;
-import javafx.scene.input.KeyCode;
-import javafx.scene.input.KeyEvent;
-import lasersharks.Direction;
-import lasersharks.interfaces.DirectionCallback;
+import java.util.Arrays;
+import java.util.Collection;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.*;
```
Comment (Exclaminator) [inline src/test/java/lasersharks/controllers/DirectionInputControllerTest.java]: Please specify imports.

File: src/main/java/lasersharksgui/panes/StandardPane.java [hunk 3]
```diff
@@ -3,14 +3,8 @@
 import javafx.scene.control.Button;
 import javafx.scene.image.Image;
 import javafx.scene.image.ImageView;
-import javafx.scene.layout.Background;
-import javafx.scene.layout.BackgroundImage;
-import javafx.scene.layout.BackgroundPosition;
-import javafx.scene.layout.BackgroundRepeat;
-import javafx.scene.layout.BackgroundSize;
-import javafx.scene.layout.Pane;
+import javafx.scene.layout.*;
```
Comment (Exclaminator) [inline src/main/java/lasersharksgui/panes/StandardPane.java]: Please specify imports.

File: src/test/java/lasersharks/LoggerTest.java [hunk 4]
```diff
@@ -1,20 +1,16 @@
 package lasersharks;
 
+import org.junit.After;
+import org.junit.Test;
+
+import java.io.*;
```
Comment (Exclaminator) [inline src/test/java/lasersharks/LoggerTest.java]: Please specify imports.

File: src/test/java/lasersharks/OptionsTest.java [hunk 5]
```diff
@@ -1,17 +1,12 @@
...
-
-import java.awt.Dimension;
-import java.awt.Toolkit;
-
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
 
-import lasersharks.Options;
+import java.awt.*;
```
Comment (Exclaminator) [inline src/test/java/lasersharks/OptionsTest.java]: Please specify imports.

File: src/test/java/lasersharks/OptionsTest.java [hunk 6]
```diff
@@ -1,17 +1,12 @@
...
+
+import static org.junit.Assert.*;
```
Comment (Exclaminator) [inline src/test/java/lasersharks/OptionsTest.java]: Please specify imports.

File: src/test/java/lasersharks/PositionTest.java [hunk 7]
```diff
@@ -1,12 +1,10 @@
 package lasersharks;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
-
 import org.junit.Before;
 import org.junit.Test;
 
+import static org.junit.Assert.*;
```
Comment (Exclaminator) [inline src/test/java/lasersharks/PositionTest.java]: Please specify imports.

File: src/test/java/lasersharks/seaobjects/LaserSharkTest.java [hunk 8]
```diff
@@ -1,24 +1,17 @@
...
-import lasersharks.seaobjects.Ammo;
-import lasersharks.seaobjects.Fish;
-import lasersharks.seaobjects.LaserShark;
-import lasersharks.seaobjects.SeaObject;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.*;
```
Comment (Exclaminator) [inline src/test/java/lasersharks/seaobjects/LaserSharkTest.java]: Please specify imports.

File: src/test/java/lasersharks/seaobjects/SeaObjectTest.java [hunk 9]
```diff
@@ -1,18 +1,14 @@
...
-import org.junit.Test;
-
 import lasersharks.Direction;
 import lasersharks.Position;
 import lasersharks.interfaces.Displayable;
-import lasersharks.seaobjects.SeaObject;
+import org.junit.After;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
```
Comment (Exclaminator) [inline src/test/java/lasersharks/seaobjects/SeaObjectTest.java]: Please specify imports.

**Comment by Exclaminator** [comment]
Looks really good! Before the merge I can only find some minor issues!

File: src/main/java/lasersharksgui/panes/StandardPane.java (hunk 3 above)
Comment (Exclaminator) [inline src/main/java/lasersharksgui/panes/StandardPane.java]: This onse still needs to be fixed ;P

Comment (MichielvdBerg) [inline src/main/java/lasersharksgui/panes/StandardPane.java]: I think I have seen to much of these imports that I overlook this one :p

**Comment by SytzeAndr** [comment]
looks good. I approve, are there still things from Youri's recommendations that are not yet implemented but should be?

**Comment by MichielvdBerg** [comment]
No, every issue  has been resolved

**Pull request merged**